src/TeamSyncer.ts

```diff
@@ -470,7 +470,7 @@ export class TeamSyncer {
                 name: `#${channel.name}`,
                 topic,
                 visibility: isPublic ? "public" : "private",
-                room_alias: alias,
+                room_alias_name: alias,
```

Comment:
I don't know how to verify this option name other than confirming that it is also used in tests of the bridge lib.
https://github.com/matrix-org/matrix-appservice-bridge/blob/ac522565ecc71d43d5cec7f972372acf92877fdd/spec/integ/bridge.spec.js#L216

Reply:
Sort of thing we should really have a type for. Wish the js-sdk exported types that we could re-use.

Reply:
For reference and extra confidence, we also use this option in the Gitter bridge, https://gitlab.com/gitterHQ/webapp/-/blob/0b9f181394767c4cff7c131f6da9ec30314daa0d/modules/matrix-bridge/lib/matrix-utils.js#L84